### PR TITLE
Fix enleak test for error-with-failure-redirect

### PR DESCRIPTION
**Pull request by denysoblohin-okta**
## Description:

Fix enleak test for `error-with-failure-redirect`
Added i18n `idx.error.code.user_not_assigned`
**! Note 1:** for `ok-PL` I've created translation manually, not with `pseudo-loc`
**! Note 2:** To prevent redirect to `error.html` (enleak should be tested before redirect) I've changed implementation of `Util.redirectWithForm` - create submit button and click it instead of calling `form.submit`. This way we can add `submit` event listener and prevent default action in test.

## PR Checklist

- [x] Have you verified the basic functionality for this change?

- [ ] Added unit tests?

- [ ] Added e2e tests

- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?

### Screenshot/Video:
Screenshot 2021-05-27 at 01 29 33
(will redirect to `http://localhost:3000/error.html?key=idx.error.code.user_not_assigned`)

### Issue:

- [OKTA-390647](https://oktainc.atlassian.net/browse/OKTA-390647)

### CODE REVIEW

File: src/util/Util.js [hunk 1]
```diff
@@ -196,7 +205,7 @@ Util.redirectWithForm = function(url, method = 'post') {
   const form = buildDynamicForm(url, method);
 
   mainContainer.appendChild(form);
-  form.submit();
+  form.querySelector('input[type="submit"]').click();
```
Comment (denysoblohin-okta) [inline src/util/Util.js:208]: To be able to prevent redirect in tests

File: packages/@okta/i18n/src/properties/login_ok_PL.properties [hunk 2]
```diff
@@ -804,3 +804,4 @@ authfactor.challenge.soft_token.invalid_passcode = 》Ýöûŕ çöðé ðöéš
 authfactor.challenge.soft_token.used_passcode = 》Éåçĥ çöðé çåñ öñļý ƀé ûšéð öñçé· Þļéåšé ŵåîţ ƒöŕ å ñéŵ çöðé åñð ţŕý åĝåîñ· 홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
 oie.tooManyRequests = 》Ţöö ɱåñý ŕéǫûéšţš 한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
 E0000010 = 》Šéŕṽéŕ îš ûñåƀļé ţö ŕéšþöñð åţ ţĥé ɱöɱéñţ· ヸ€홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
+idx.error.code.user_not_assigned = 》Ýöû åŕé ñöţ åļļöŵéð ţö åççéšš ţĥîš ţĥîš åþþ· Ţö ŕéǫûéšţ åççéšš، çöñţåçţ åñ åðɱîñ· 한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
```
Comment (denysoblohin-okta) [inline packages/@okta/i18n/src/properties/login_ok_PL.properties:807]: This string was manually created, not generated with pseudo-loc

Comment (nikhilvenkatraman-okta) [reply packages/@okta/i18n/src/properties/login_ok_PL.properties:879]: @denysoblohin-okta curious why we had to generate this manually ?  There was a bug in the script that I fixed recently https://github.com/okta/okta-signin-widget/pull/2000

File: .eslintrc.js [hunk 3]
```diff
@@ -100,7 +100,6 @@ module.exports = {
         {
           'files': [
             'playground/**/authenticator-verification-okta-verify-reject-push.json',
-            'playground/**/error-with-failure-redirect.json',
```
Comment (nikhilvenkatraman-okta) [inline .eslintrc.js:103]: Might get some merge conflict as I made a recent change to clean up this list https://github.com/okta/okta-signin-widget/pull/1999

File: src/util/Util.js [hunk 4]
```diff
@@ -28,6 +28,13 @@ const buildInputForParameter = function(name, value) {
   return input;
 };
 
+const buildSubmitInput = function() {
+  const input = document.createElement('input');
+  input.type = 'submit';
+  input.value = 'Submit';
```
Comment (nikhilvenkatraman-okta) [inline src/util/Util.js:34]: Avoid using hard-coded english text

File: packages/@okta/i18n/src/properties/login_ok_PL.properties (hunk 2 above)
Comment (denysoblohin-okta) [reply packages/@okta/i18n/src/properties/login_ok_PL.properties:880]: I can't install `pseudo-loc` with `yarn install`. 
I get error `https://artifacts.aue1d.saasure.com:443/artifactory/api/npm/npm-okta-all/@okta/app-localizer/-/@okta/app-localizer-1.2.3-105-g97aab49.tgz: getaddrinfo ENOTFOUND artifacts.aue1d.saasure.com`

Comment (nikhilvenkatraman-okta) [reply packages/@okta/i18n/src/properties/login_ok_PL.properties:880]: Yes, this requires VPN as it is an internal package

File: test/testcafe/spec-en-leaks/EnglishLeaks_spec.js [hunk 5]
```diff
@@ -92,6 +91,9 @@ const setUpResponse = (filePath) => {
 async function setup(t, locale) {
   const widgetView = new PageObject(t);
   await widgetView.navigateToPage();
+  await widgetView.preventRedirect([
```
[the remark this answers is not in the document]
Comment (denysoblohin-okta) [reply test/testcafe/spec-en-leaks/EnglishLeaks_spec.js]: Done

File: package.json [hunk 6]
```diff
@@ -153,7 +153,7 @@
     "handlebars": "^4.5.3",
     "q": "1.4.1",
     "u2f-api-polyfill": "0.4.3",
-    "underscore": "1.8.3"
+    "underscore": "1.13.1"
```
Comment (nikhilvenkatraman-okta) [inline package.json:156]: This change is already merged https://github.com/okta/okta-signin-widget/pull/2007/files. Can you revert this on your PR and base your changes off the `5.7` branch and rebase ?

File: packages/@okta/i18n/src/properties/login_ok_PL.properties (hunk 2 above)
Comment (nikhilvenkatraman-okta) [reply packages/@okta/i18n/src/properties/login_ok_PL.properties:880]: @denysoblohin-okta  what you have generated manually seems fine. If you need help generating this using the script let me know and I can checkout your branch and run the script.

File: test/testcafe/spec-en-leaks/EnglishLeaks_spec.js [hunk 7]
```diff
@@ -19,10 +19,15 @@ const ignoredMocks = [
   'error-internal-server-error.json',
   'error-forgot-password.json',
   'authenticator-verification-select-authenticator.json',
+  'authenticator-verification-okta-verify-signed-nonce-loopback.json',
```
Comment (nikhilvenkatraman-okta) [inline test/testcafe/spec-en-leaks/EnglishLeaks_spec.js:22]: Why are we adding this back to the ignore list ?

File: package.json (hunk 6 above)
Comment (denysoblohin-okta) [reply package.json]: The branch was already rebased on 5.7, but base branch in GitHub was master. Fixed

File: test/testcafe/spec-en-leaks/EnglishLeaks_spec.js (hunk 7 above)
Comment (denysoblohin-okta) [reply test/testcafe/spec-en-leaks/EnglishLeaks_spec.js]: Fxed

**Review by nikhilvenkatraman-okta — APPROVED** [review]

**Comment by denysoblohin-okta** [comment]
Merged to 5.7

**Pull request closed without merging**